core/src/main/scala/org/http4s/Headers.scala

```diff
@@ -95,6 +96,14 @@ final class Headers private (headers: List[Header])
       this.toList.equals(otherheaders.toList)
     case _ => false
   }
+
+  /** Removes the `Content-Length`, `Content-Range`, `Trailer`, and
```

Comment:
This PR contains both two indentation and single indentation function annotation style. Should probably decide which style we prefer, or atleast stay consistent within a PR.